k9mail/src/main/java/com/fsck/k9/activity/compose/AttachmentPresenter.java

```diff
@@ -345,6 +389,12 @@ public void onClickRemoveAttachment(Uri uri) {
         listener.onAttachmentRemoved();
     }
 
+    public void updateAttachmentsList(Attachment attachment){
+        Attachment originalAttachment = attachments.get(attachment.uri);
+        originalAttachment.updateResizeInfo(attachment.resizeFactor, attachment.overrideDefault);
+        Log.d("ATTACH","Yes : " + attachments.size());
```

Comment:
Stick to K9.LOG_TAG and change the logging to be more useful.

Also, you'll need to rebase this code to use `Timber` sooner or later.